homeassistant/components/media_player/plex.py

```diff
@@ -272,93 +527,383 @@ def _active_media_plexapi_type(self):
     @property
     def media_content_id(self):
         """Content ID of current playing media."""
-        if self.session is not None:
+        if self.session:
             return self._convert_na_to_none(self.session.ratingKey)
 
     @property
     def media_content_type(self):
         """Content type of current playing media."""
-        if self.session is None:
+        if self.state not in (STATE_IDLE, STATE_OFF):  # true or frozen state
```

Comment:
We also seem to check these two statements regularly. Would it make sense to put it in a property  ? Or, just resolve all these things inside the `update` method.